mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -22,6 +22,8 @@
      then we will show another collapsed entry saying "Access your data on 2 other domains". This entry it's for the plural case, when the add-on is accessing more than one extra domain.
      %1$d will be replaced by an integer indicating the number of additional domains for which this web extension is requesting permission. -->
     <string name="mozac_feature_addons_permissions_extra_domains_description_plural" tools:ignore="PluralsCandidate">Access your data on %1$d other domains</string>
+    <!-- Content description for a permission item, for an installed add-on details page. %1$s will be replaced with the permission text, %2$d will be replaced with the permission position in the list and %3$d will be replaced with the total number of permissions in the list. the whole text will read permission, 2 of 6. -->
```

Comment:
@Amejia481: Is there a reason we are not hardcoding "Permissions" here?
In the meantime: can you clarify what the string ID is for the "permission" string atm?

Reply:
To clarify, permission can be one these [string IDs](https://github.com/mozilla-mobile/firefox-android/blob/9cb30bc35e0d1006cd8d6272d820452d8f002209/android-components/components/feature/addons/src/main/res/values/strings.xml#L7-L77). Taking as an example [mozac_feature_addons_permissions_privacy_description](https://github.com/mozilla-mobile/firefox-android/blob/9cb30bc35e0d1006cd8d6272d820452d8f002209/android-components/components/feature/addons/src/main/res/values/strings.xml#L7), the text will read  as **"Read and modify privacy settings, 2 of 6"**.

Reply:
Thanks, could you clarify that in comments? Right now I'm not sure localizers will get the full picture and might get confused (and since the string is mostly placeholders, that could lead to awkward tranlations)

Reply:
Sure, I'll update it :)

Reply:
I filed this [PR](https://github.com/mozilla-mobile/firefox-android/pull/5388) for it :)